src/components/StaticFilters.tsx

```diff
@@ -21,23 +13,15 @@ export interface StaticFiltersCssClasses {
   optionsContainer?: string,
   option?: string,
   optionInput?: string,
-  optionLabel?: string,
-  applyFiltersButton?: string
+  optionLabel?: string
 }
 
 /**
- * The configuration data for a filter option.
+ * The configuration data for a static filter option.
  *
  * @public
  */
-export interface FilterOptionConfig {
-  /** {@inheritDoc Filters.CheckboxOptionProps.value} */
-  value: string | number | boolean,
-  /** {@inheritDoc Filters.CheckboxOptionProps.label} */
-  label?: string,
-  /** {@inheritDoc Filters.CheckboxOptionProps.selectedByDefault} */
-  selectedByDefault?: boolean
-}
+export type StaticFilterOptionConfig = Omit<FilterOptionConfig, 'matcher'>;
```

Comment:
since we're only accepting `$eq` matcher, then we also need to limit the `value` to not accept `NumberRangeValue`.

but, it might also just be easier to accept any matcher and then we can probably just use `FilterOptionConfig` here. I'm not sure if we've tested that for static filters though

Reply:
The discussion thread on StaticFilters with product seems to settle on accepting $eq only so we can keep it as is. I will limit `value` to omit numberRangeValue